### ISSUE TITLE
devel-project: automatically prune devel-whitelist once included as devel project

### ISSUE THREAD

**Issue by jberry-suse** [openSUSE/osc-plugin-factory#1018]
Seems reasonable to automatically remove projects from `devel-whitelist` in remote config (#1014) once they are elevated to a devel project.

**Comment by jberry-suse** [comment]
Would seem to make sense to just make it part of `list --write` otherwise perhaps add `--prune` option, but that should be included in nightly run on packagelists.

**Comment by akshat5302** [comment]
Is this issue is open??
I would like to contribute to it if open please elaborate on it...

**Comment by coolo** [comment]
Yeah, the issue is open. But 'less complex' is mostly relative - for someone understanding the full context, not really for initial contributors.

**Comment by akshat5302** [comment]
From where to start I'm interested @coolo

**Comment by coolo** [comment]
Well, this is what I mean - Jimmy labeled it "less complex", but meant for himself :)

I don't even think it's possible for a bot to write the OSRT:Config attribute. But this is what the issue is asking for: compare the devel project whitelist in the config with the existing devel projects (calculcated in `list` mode) and if the whitelist contains an existing project, erase it from the config's white list.

I'll remove the tag

**Comment by akshat5302** [comment]
Ok @coolo
